### PR TITLE
Fix missing i_suppress_dialog parameter syntax error in 731 system

### PR DESCRIPTION
**Pull request by sercankd**
Referenced to #3426

i_suppress_dialog parameter is also missing at **save** method of 'if_package' interface in 731 system i've used. It gives syntax error upon activation.

### CODE REVIEW

File: src/objects/zcl_abapgit_object_devc.clas.abap [hunk 1]
```diff
@@ -419,18 +419,32 @@ CLASS zcl_abapgit_object_devc IMPLEMENTATION.
...
+            EXCEPTIONS
+              object_invalid        = 1
+              object_not_changeable = 2
+              cancelled_in_corr     = 3
+              permission_failure    = 4
+              unexpected_error      = 5
+              intern_err            = 6
+              OTHERS                = 7.
 
+        CATCH cx_root.
```
Comment (mbtools) [inline src/objects/zcl_abapgit_object_devc.clas.abap:435]: let's use `CATCH cx_sy_dyn_call_param_not_found.`

Comment (mbtools) [reply src/objects/zcl_abapgit_object_devc.clas.abap:435]: if you don't mind, could you change the other catch cx_root in this class, too? thanks

Comment (sercankd) [reply src/objects/zcl_abapgit_object_devc.clas.abap]: @mbtools Yes, i did but checks have failed.

Comment (sercankd) [reply src/objects/zcl_abapgit_object_devc.clas.abap]: nvm my bad, i put an extra single quote by mistake. should be fine now.

**Review by mbtools — APPROVED** [review]
Thank you 👍

**Comment by larshp** [comment]
welcome to abapGit as a contributor 🎉

**Pull request merged**